src/config/vim.zig

```diff
@@ -88,6 +88,7 @@ fn writeSyntax(writer: anytype) !void {
         \\let s:cpo_save = &cpo
         \\set cpo&vim
         \\
+        \\syn iskeyword @,-
```

Comment:
Some config keywords have digits, so this should include digits as well.

```suggestion
        \\syn iskeyword @,48-57,-
```